src/app/components/feed/FeedLastClusterizedAt.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { IntlProvider } from 'react-intl';
+import { FeedLastClusterizedAt } from './FeedLastClusterizedAt';
+
+describe('<FeedLastClusterizedAt />', () => {
+  it('should render FormattedDate when last_clusterized_at ', () => {
```

Comment:
I think the test name here is incomplete - probably you meant something like "should render FormattedDate when last_clusterized_at is not null"?